concur/integrations/glfw.py

```diff
@@ -107,6 +117,7 @@ def main(
         screencast_fps: Save the screencast video with a given FPS.
         menu_bar: Reserve space for `concur.widgets.main_menu_bar` at the top of the window.
         maximized: Create a maximized window.
+        is_retina: Fix blurred fonts on retina displays
```

Comment:
The Retina fix is behind an option, because it doesn't work with default font? If so, could you add this information into this docstring?

Reply:
Im not sure what you mean by 'because it doesn't work with default font'? I got the fix from here:
https://pyimgui.readthedocs.io/en/latest/guide/using-fonts.html?highlight=retina#high-density-screens

Reply:
In the linked text:

> If you set global scaling factor then you should also probably clear whole font atlas before adding any font atlas using _FontAtlas.clear(). Otherwise **default built in font** that is loaded on imgui startpup **will be too small to read**. Its place will be taken by first font you add to the atlas.

So the retina fix presumably doesn't work properly with the default font. Otherwise, what would be the reason to ever use `is_retina=False`?

Reply:
Hmm, I thought to give the dev an opportunity to switch it on or off but I have it on all the time and am not noticing any problems only improvements, so maybe we could remove the flag indeed and put it in by default.